torch/csrc/jit/script/module.h

```diff
@@ -461,6 +457,10 @@ struct Module {
     return parameters[name].slot();
   }
 
+  const NamedIValue* get_named_parameter(const std::string& name) const {
+    return parameters.find(name);
```

Comment:
Is the intention that this will only ever be called with names that won't fail? Otherwise maybe error here